src/method/query_method.cpp

```diff
@@ -569,3 +583,39 @@ method_fixup_vobjs (DB_VALUE *value_p)
 
   return rc;
 }
+#endif
+
+#if defined (SERVER_MODE) || defined (SA_MODE)
+/*
+ *  xmethod_invoke_fold_constants () - perform constant folding for method
+ *  return	  : error code
+ *  thread_p (in) : worker thread
+ *  sig_list(in) : method signature
+ *  args(in) : method argument
+ */
+int xmethod_invoke_fold_constants (THREAD_ENTRY *thread_p, const method_sig_list &sig_list,
+				   std::vector<std::reference_wrapper<DB_VALUE>> &args,
+				   DB_VALUE &result)
+{
+  int error_code = NO_ERROR;
+  cubmethod::method_invoke_group method_group (thread_p, sig_list);
+  method_group.begin ();
+  error_code = method_group.prepare (args);
+  if (error_code != NO_ERROR)
+    {
+      return error_code;
```

Comment:
It looks like you need to call method_group.end() before returning.

Reply:
Thank you. I've added to call end() at the destructor of method_invoke_group.